### DISCUSSION TITLE
Price Chart change

### DISCUSSION THREAD

**Discussion by stevenstallone** [morning4coffe-dev/project-sbs#117] [Ideas]
The app has a history price chart for 6 months. I don't see how that is entirely useful to see what the price of what was was paid the previous 6 months. I did notice a spike in July and realized that next year I wouldn't see it coming because the price chart is the previous 6 months. Subscriptions are typically a constant, so the chart should reflect what should be paid for the whole year. if the price chart just displayed a graph for all 12 months, you could easily see that there is a spike in price coming up. You don't have this when it's simply historical. You can then easily gauge what the subscription year is like for a year.

**Comment by morning4coffe-dev** [comment]
@stevenstallone Thank you for the feedback! I see what you're getting at, and you're right—subscriptions are generally stable, so it would be helpful to have a clearer view of the entire year's pricing rather than just historical data. A full-year overview would definitely make it easier to spot upcoming price changes.

I’ll consider adding this feature in a future version. Currently, I’m focused on adding multiplatform support and preparing for a release to multiple stores. You can follow the progress here: https://github.com/morning4coffe-dev/recurrents/pull/116.
